app/views/govuk_publishing_components/components/_details.html.erb

```diff
@@ -6,15 +6,18 @@
   css_classes = %w(gem-c-details govuk-details)
   css_classes << (shared_helper.get_margin_bottom)
 
+  details_data_attributes = {}
+  details_data_attributes[:module] = 'govuk-details gem-details'
+
   data_attributes ||= {}
-  data_attributes[:module] = 'govuk-details gem-details'
+  data_attributes[:details_track_click] = ''
 %>
-<%= tag.details class: css_classes, data: data_attributes, open: open do %>
-  <summary class="govuk-details__summary" data-details-track-click>
+<%= tag.details class: css_classes, data: details_data_attributes, open: open do %>
+  <%= tag.summary class: "govuk-details__summary", data: data_attributes do %>
```

Comment:
If we've got `details_data_attributes`, might it be worth renaming `data_attributes` to `summary_data_attributes`?
```suggestion
  <%= tag.summary class: "govuk-details__summary", data: summary_data_attributes do %>
```

Reply:
I think I kept it like that because `data_attributes` is something that can be passed to the component, and changing it would mean a breaking change to the interface. But happy to be corrected if I've misunderstood.

Reply:
Oh yeah, that's correct 🤦 Best kept as `data_attributes` then.